src/Endpoints/class-analytics-post-detail-api-proxy.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Endpoints: Parse.ly `/analytics/post/detail` API proxy endpoint class
+ *
+ * @package Parsely
+ * @since   3.6.0
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\Endpoints;
+
+use stdClass;
+use WP_REST_Request;
+
+/**
+ * Configures the `/analytics/post/detail` REST API endpoint.
+ */
+final class Analytics_Post_Detail_API_Proxy extends Base_API_Proxy {
+
+	/**
+	 * Registers the endpoint's WP REST route.
+	 */
+	public function run(): void {
+		$this->register_endpoint( '/analytics/post/detail' );
+	}
+
+	/**
+	 * Cached "proxy" to the Parse.ly `/analytics/post/detail` API endpoint.
+	 *
+	 * @param WP_REST_Request $request The request object.
+	 */
+	public function get_items( WP_REST_Request $request ): stdClass {
+		return $this->get_data( $request );
+	}
+
+	/**
+	 * Generates the final data from the passed response.
+	 *
+	 * @param array<string, mixed> $response The response received by the proxy.
+	 * @return array<stdClass> The generated data.
+	 */
+	protected function generate_data( array $response ): array {
+		$stats_base_url = trailingslashit( 'https://dash.parsely.com/' . $this->parsely->get_api_key() ) . 'find';
```

Comment:
Minor suggestion for another ticket - consider having `dash.parsely.com` defined in a constant that could then be adjusted during development to test against the beta dashboard.

Reply:
We are defining this in 2-3 places within our codebase. Noting this to see if it must be a function or constant.

Reply:
A function would be fine too, or a public constant/method in a class that could then be mocked / overridden locally.

Reply:
Now tracked in #1208.